packages/create/bin/dependencies.ts

```diff
@@ -23,6 +23,12 @@ export function normaliseForwardedArgv(
     const updatedForwardArgv = forwardArgv;
     updatedForwardArgv['preset'] =
         forwardArgv['preset'] === 'next' ? 'apps' : forwardArgv['preset'];
+
+    // As playwright is not supported by the Nx generator for e2eTestRunner, we pass none to proceed and later install it ourselves as part of Stacks Playwright plugin.
+    updatedForwardArgv['e2eTestRunner'] =
+        updatedForwardArgv['e2eTestRunner'] === 'playwright'
+            ? 'none'
+            : updatedForwardArgv['e2eTestRunner'];
```

Comment:
We need to do this for both Playwright and Cypress, forwarding Cypress to the Nx create script would then cause errors when it comes to calling our own Cypress plugin

Reply:
ah true. Well makes the fix simpler :)